book/src/external-health-monitor-controller.md

```diff
@@ -9,11 +9,12 @@
 ### Supported Versions
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image
 --|--|--|--|--
-[external-health-monitor-controller v0.3.0](https://github.com/kubernetes-csi/external-health-monitor/releases/tag/v0.3.0) | [release-0.3](https://github.com/kubernetes-csi/external-health-monitor/tree/release-0.3) | [v1.3.0](https://github.com/container-storage-interface/spec/releases/tag/v1.3.0) | - | k8s.gcr.io/sig-storage/csi-external-health-monitor-controller:v0.3.0
+[external-health-monitor-controller v0.4.0](https://github.com/kubernetes-csi/external-health-monitor/releases/tag/v0.4.0) | [release-0.4](https://github.com/kubernetes-csi/external-health-monitor/tree/release-0.4) | [v1.4.0](https://github.com/container-storage-interface/spec/releases/tag/v1.5.0) | - | k8s.gcr.io/sig-storage/csi-external-health-monitor-controller:v0.4.0
 
 ### Unsupported Versions
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image
 --|--|--|--|--
+[external-health-monitor-controller v0.3.0](https://github.com/kubernetes-csi/external-health-monitor/releases/tag/v0.3.0) | [release-0.3](https://github.com/kubernetes-csi/external-health-monitor/tree/release-0.3) | [v1.3.0](https://github.com/container-storage-interface/spec/releases/tag/v1.3.0) | - | k8s.gcr.io/sig-storage/csi-external-health-monitor-controller:v0.3.0
```

Comment:
This version is not a year old yet, so we're still supporting it

Reply:
@msau42 is our policy for supporting a version for an year from its release or its something like last a couple or 3/4 releases will be supported? regardless I am addressing the comments. Just thought of clarifying it.

Reply:
The policy is one year: https://kubernetes-csi.github.io/docs/project-policies.html#support

Reply:
Thanks!